### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by serg121192**

### CODE REVIEW

File: app/cafe.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import datetime
+
+
+from app.errors import NotVaccinatedError
+from app.errors import NotWearingMaskError
+from app.errors import OutdatedVaccineError
```
Comment (olena-hul) [inline app/cafe.py:6]: You can use 1 import for this

File: app/cafe.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor['name']} must be vaccinated!")
+
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Vaccine should be fresh!")
```
Comment (olena-hul) [inline app/cafe.py:19]: Nit: fresh is related rather to food than vaccines) You can say that "The vaccine has expired" or something like this)

**Review by olena-hul — CHANGES_REQUESTED** [review]

File: app/cafe.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor['name']} must be vaccinated!")
+
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Your vaccine has expired!")
+
+        if visitor["wearing_a_mask"] is False:
```
Comment (ArsenPidhoretskyi) [inline app/cafe.py:23]: use .get("wearing_a_mask") possible wearing_a_mask can not exist in visitor. Also you can write as:
```suggestion
        if not visitor.get("wearing_a_mask"):

```

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Review by ArsenPidhoretskyi — APPROVED** [review]
Well done!

**Pull request closed without merging**